cswinrt/main.cpp

```diff
@@ -31,6 +31,7 @@ namespace cswinrt
         { "output", 0, 1, "<path>", "Location of generated projection" },
         { "include", 0, option::no_max, "<prefix>", "One or more prefixes to include in projection" },
         { "exclude", 0, option::no_max, "<prefix>", "One or more prefixes to exclude from projection" },
+        { "target", 0, 1, "<tfm>", "Target TFM for projection. Leave empty for code compatible with newest supported TFM. Currently supported: netstandard2.0, net5.0." },
```

Comment:
how about:
        { "target", 0, 1, "<netstandard2.0 | net5.0>", "Target for projection, defaulting to the latest available TFM" },